torchserve/assets/monitors/error_ratio.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "TorchServe: Requests error ratio is high",
+	"type": "query alert",
+	"query": "sum(last_1h):(sum:torchserve.openmetrics.requests.4xx.count{*}.as_count() + sum:torchserve.openmetrics.requests.5xx.count{*}.as_count()) / (sum:torchserve.openmetrics.requests.2xx.count{*}.as_count() + sum:torchserve.openmetrics.requests.4xx.count{*}.as_count() + sum:torchserve.openmetrics.requests.5xx.count{*}.as_count()) > 0.1",
```

Comment:
Is the error ratio always [error / (success + errors)]? 

Including redirects (3xx) and informational (1xx) responses will make the error ratio as [errored requests / all requests].

Reply:
There's no 1xx or 3xx exposed by TorchServe